logos/README.md

```diff
@@ -0,0 +1,10 @@
+# UGM logos
+Cat-themed logos for the 2023 MDAnalysis UGM, created by Fiona Naughton. Copyright (c) 2023 Fiona Naughton.
+
+ - `mdanalysisUGM_catlogo_textless.png`: image only version
+ - `mdanalysisUGM_catlogo_text`: version with text
+
+## License
+Licensed under a Creative Commons [Attribution-ShareAlike 4.0 International (CC BY-SA 4.0)](https://creativecommons.org/licenses/by-sa/4.0/) License.
+
+The original MDAnalysis 'Atom' logo was created by Christian Beckstein and is Copyright (c) 2011 Christian Beckstein (@beckstein) 
```

Comment:
MDAnalysis holds all the rights on the logo so just acknowledgment of creator but no rights implications:
```suggestion
The original MDAnalysis 'Atom' logo was created by Christian Beckstein (@beckstein).
```
We also have to change things in the branding repo.